training/src/app/modules/chuck-norris/components/chuck-norris-main/chuck-norris-main.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../../../../styles";
+
+.top-bar {
+  position: relative;
+  display: grid;
+  height: fit-content;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 12%;
+
+  #top-bar-fire {
+    position: relative;
+    width: 100%;
+    height: 100%;
+  }
+
+  #top-bar-logo {
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.11));
+    border-radius: 142.5px;
+    position: absolute;
+    width: 30%;
+    height: 65%;
+    margin-left: 35%;
+    margin-top: 38%;
+  }
+
+  #top-bar-title {
```

Comment:
we talked about centering/align items with flexbox
please use it.
margin of 34.3 etc is not valid